docker-compose.yml

```
@@ -0,0 +1,28 @@
+services:
+  db:
+    image: mongo:4.2
+    restart: always
+    container_name: six-cities_mongodb
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: ${DB_USERNAME}
+      MONGO_INITDB_ROOT_PASSWORD: ${DB_PASSWORD}
+    ports:
+      - ${DB_PORT}:27017
+    volumes:
+      - six-cities_data:/data/db
+
+  db_ui:
+    image: mongo-express:latest
+    restart: always
+    container_name: six-cities_mongo_express
+    ports:
+      - "8081:8081"
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: ${ME_CONFIG_MONGODB_ADMINUSERNAME}
+      ME_CONFIG_MONGODB_ADMINPASSWORD: ${ME_CONFIG_MONGODB_ADMINPASSWORD}
+      ME_CONFIG_MONGODB_URL: mongodb://admin:test@db:27017/
```

Лучше использовать env переменные для username и password. Безопаснее

```suggestion
      ME_CONFIG_MONGODB_URL: mongodb://${DB_USERNAME}:${DB_PASSWORD}@db:27017/
```